R/derive_param_clinbenefit.R

```diff
@@ -94,6 +102,11 @@
 #'
 #' *Default:* `yn_to_numeric` (see `admiral::yn_to_numeric()` for details)
 #'
+#' @param clinben_vals A vector of response values to be considered when determining
+#' clinical benefit.
+#'
+#' *Default:* `CR`,`PR`, `SD`, and `NON-CR/NON-PD`
```

Comment:
We have decided to remove the "Default" section for now until we find a solution to automate it. So please remove it.

Reply:
Updated